lib/main.dart

```diff
@@ -17,55 +16,143 @@ class Home extends StatelessWidget {
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: <Widget>[
+                        Text(
+                          "Next , Zohor",
+                          style: TextStyle(color: Colors.white, fontSize: 14),
+                        ),
+                        Text(
+                          '1:04 PM',
+                          style: TextStyle(color: Colors.white, fontSize: 32),
+                        ),
+                        Text(
+                          'Starts in 02:29:46',
+                          style: TextStyle(color: Colors.white, fontSize: 14),
+                        ),
+                      ],
                     ),
+                    Image.asset('assets/prayer_icons.png'),
+                  ],
+                ),
+                Spacer(flex: 16),
+                Row(
+                  children: [
+                    Image.asset('assets/calendar.png'),
+                    Text(' 9 December 2020,/23 Rhabiul Akhir 144 H',
+                        style: TextStyle(color: Colors.white, fontSize: 14)),
+                  ],
+                ),
+                Spacer(flex: 4),
+                Row(
+                  children: [
+                    Image.asset('assets/location.png'),
+                    Text(" Kuala Lumpur",
+                        style: TextStyle(color: Colors.white, fontSize: 14)),
                   ],
                 ),
-                Image.asset('assets/prayer_icons.png'),
-              ],
-            ),
-            Spacer(flex: 16),
-            Row(
-              children: [
-                Image.asset('assets/calendar.png'),
-                Text(' 9 December 2020,/23 Rhabiul Akhir 144 H',
-                    style: TextStyle(color: Colors.white, fontSize: 14)),
-              ],
-            ),
-            Spacer(flex: 4),
-            Row(
-              children: [
-                Image.asset('assets/location.png'),
-                Text(" Kuala Lumpur",
-                    style: TextStyle(color: Colors.white, fontSize: 14)),
               ],
             ),
-          ],
-        ),
+          ),
+          Prayer(),
+        ],
+      ),
+    );
+  }
+}
+
+class Prayer extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      constraints: BoxConstraints.expand(width: 375, height: 235),
+      decoration: BoxDecoration(color: Colors.white12),
+      padding: new EdgeInsets.all(16.0),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Column(
+                children: [
+                  Image.asset('assets/prayer.png'),
+                  Text("Prayer",
+                      style: TextStyle(color: Colors.black, fontSize: 14)),
```

Comment:
Please don't repeat the codes. Move this to generic method and call it here. If you have any doubt, you can ping me